### PR TITLE
plugin(apps-visualizer): live stats

### PR DESCRIPTION
**Pull request by leosmerling-hopeit**
Adding live stats to apps-visualizer, based on activity from log stream.

Combining `log-streamer` with `apps-visualizer` plugins, live activity
visualization of events execution and stream processing is possible since version 0.6.0.

### CODE REVIEW

**Review by pcanto-hopeit — APPROVED** [review]

File: apps/examples/simple-example/api/openapi.json [hunk 1]
```diff
@@ -1456,6 +1585,595 @@
...
+      },
+      "APIConfig": {
+        "type": "object",
+        "properties": {
+          "docs_path": {
+            "type": "string"
+          }
+        },
+        "x-module-name": "hopeit.server.config",
+        "description": "APIConfig(docs_path: Union[str, NoneType] = None)"
```
Comment (pcanto-hopeit) [inline apps/examples/simple-example/api/openapi.json:2158]: this line should be:
 "description": "APIConfig(docs_path: Optional[str] = None)"

**Review by pcanto-hopeit — CHANGES_REQUESTED** [review]
openapi file `apps/examples/simple-example/api/openapi.json` should be updated, also
`engine/config/schemas/[app-config-schema-draftv6.json,server-config-schema-draftv6.json]`

File: apps/examples/simple-example/api/openapi.json (hunk 1 above)
Comment (leosmerling-hopeit) [reply apps/examples/simple-example/api/openapi.json:2158]: Optional[str] is actually an alias for Union[str, NoneType]

**Comment by leosmerling-hopeit** [comment]
> openapi file `apps/examples/simple-example/api/openapi.json` should be updated, also
> `engine/config/schemas/[app-config-schema-draftv6.json,server-config-schema-draftv6.json]`

API file was already updated using `source apps/examples/simple-example/api/create-openapi-file.sh`, no changes after rerun.

Schemas updated in f4ce3e59fe5bcc53ac76e8ff2bb229ee3dfe7b87q

**Review by pcanto-hopeit — APPROVED** [review]

**Pull request merged**